semperfiwebdesign/all-in-one-seo-pack
# Add filter to remove noindex for RSS feeds

Issue #2215

## Proposed changes

This adds a filter to remove the noindex HTTP header for RSS feeds.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist


## Testing instructions

Test the filter with the code example in the documentation (and then also publish the documentation) - https://semperplugins.com/wp-admin/post.php?post=4982&action=edit&lang=en and the following URL: www.examplesite.org/feed. You'll need to check the HTTP headers through Chrome's Dev Tools.
